src/test/java/org/inferred/freebuilder/processor/RequiredPropertiesTest.java

```diff
@@ -248,6 +302,129 @@ public void testBuilderGetterThrowsIfValueUnset() {
         .runTest();
   }
 
+  @Test
+  public void testMergeFromBuilder_defaultsDoNotOverride() {
```

Comment:
They are copies! But `DEFAULTED_PROPERTIES_TYPE` is different — it disables a javac optimization, so we can verify the behaviour is not affected. (And, technically, this uses the "property is required" code path, even though the property is defaulted, hence why it's ended up in this test case.)

On reflection, this is rather confusing and likely to get out-of-sync, so I'll do some JUnit magic to parameterize DefaultedPropertiesTest instead. Thanks!